src/services/firebase-app.js

```diff
@@ -0,0 +1,12 @@
+import firebase from "firebase/app";
+
+// Initialize Firebase
+const config = {
+  apiKey: "AIzaSyCGpm5s4Fvwr40RLpEpjI_Wa1xksPSJb9s",
```

Comment:
you should not push keys to the repository. Moreover database configuration is not a part of the hosting so why is it here?

Reply:
Before using hosting I needed to initialize our app. this `config` is recommended by Firebase developers. They also ensure, that this key in public doesn't affect on app and user's data security. I tried to deploy app without key, and it worked ok for me, but it might affect on other scope. Should I remove it?
![image](https://user-images.githubusercontent.com/14272396/38353902-9062237a-38c0-11e8-880d-af0c900fa389.png)
https://firebase.google.com/docs/web/setup?hl=ru#add_firebase_to_your_app

Reply:
how does this relates to hosting?

Reply:
what do you mean how? I cant host not initialized app

Reply:
it is not database configuration. it is firebase configuration. it is common for database, authentication, storage etc.

Reply:
when I initialized firebase, I passed questionnaire about what features was I going to use. So I choose database and hosting straight.